test/ci_support/example_test_failure_github_issue.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+
+#
+# Implementation code
+#
+
+# Find the implementation
+tribitsDir = os.environ.get("TRIBITS_DIR", "")
+if tribitsDir:
+  sys.path = [os.path.join(tribitsDir, "ci_support")] + sys.path
+else:
+  raise Exception("ERROR, TRIBITS_DIR must be set in the env to 'tribits' base dir!")
+
+import CreateIssueTrackerFromCDashQuery as CITFCQ
+
+
+usageHelp = \
+r"""Stock error message.
+"""
+
+
+# The main function
+def main():
+  issueTrackerCreator = \
+    CITFCQ.CreateIssueTrackerFromCDashQueryDriver(
+      ExampleIssueTrackerFormatter(),
+      cdashProjectStartTimeUtc="6:00", # Midnight MST
+      usageHelp=usageHelp,
+      issueTrackerUrlTemplate="https://github.com/<group>/<group>/issues/<newissueid>",
```

Comment:
`<group>/<group>`?  Should the second one be something like `<repo>`?

Reply:
Good catch.  I have this fixed in my local branch.